src/content/1.7/modules/concepts/hooks/list-of-hooks.md

```diff
@@ -1206,6 +1208,8 @@ This hook launches modules when the AdminOrder tab is displayed in the Back Offi
 
     Located in: /controllers/admin/AdminOrdersController.php
 
+    Replaced by **displayAdminOrderTabContent** in 1.7.7
```

Comment:
I think it's probably best to note this at the top of the description instead at the bottom.

See line 303 for an example of how to do it.